modular_skyrat/modules/admin/code/sooc.dm

```diff
@@ -16,7 +16,7 @@ GLOBAL_LIST_EMPTY(ckey_to_sooc_name)
 	if(!mob)
 		return
 
-	var/static/list/job_lookup = list(JOB_SECURITY_OFFICER=TRUE, JOB_WARDEN=TRUE, JOB_DETECTIVE=TRUE, JOB_HEAD_OF_SECURITY=TRUE, JOB_CAPTAIN=TRUE, JOB_BLUESHIELD=TRUE, JOB_SECURITY_MEDIC=TRUE, JOB_SECURITY_SERGEANT=TRUE, JOB_CIVIL_DISPUTES_OFFICER=TRUE)
+	var/static/list/job_lookup = list(JOB_CAPTAIN=TRUE, JOB_HEAD_OF_SECURITY=TRUE, JOB_WARDEN=TRUE, JOB_DETECTIVE=TRUE, JOB_SECURITY_OFFICER=TRUE, JOB_SECURITY_OFFICER_MEDICAL=TRUE, JOB_SECURITY_OFFICER_ENGINEERING=TRUE, JOB_SECURITY_OFFICER_SCIENCE=TRUE, JOB_SECURITY_OFFICER_SUPPLY=TRUE, JOB_CORRECTIONS_OFFICER=TRUE, JOB_SECURITY_MEDIC=TRUE, JOB_SECURITY_SERGEANT=TRUE, JOB_CIVIL_DISPUTES_OFFICER=TRUE, JOB_BLUESHIELD=TRUE)
```

Comment:
We don't use all those security officer variants, we use departmental guards instead.

Reply:
Yes, I left them because they're still defined in jobs.dm even if we're not using them at the moment. I'll add the guards, want me to remove the inactive jobs?